### PR TITLE
feat: t01 show total jobs and t02 show first 100 jobs

### PR DESCRIPTION
**Pull request by TanDangNgoc**
Note:
1. Using bootstrap for styling header
2. Call API from mockAPI and show the total numbers of jobs are available
3. Link surge: ashamed-advertisement.surge.sh
4. Unit Test (First time writing unit test)
image

### CODE REVIEW

**Review by hienntfaba — CHANGES_REQUESTED** [review]

File: src/containers/Home/component/SearchBox/index.tsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+import './searchbox.scss'
+
+interface Props {
+    availableJobs: number;
+    totalJobs: number;
+}
+
+const Content: React.FC<Props> = ({availableJobs, totalJobs}) => {
+    return (
+        <div className='searching_box bg-dark text-white container-fluid' data-testid='searching_box'>
```
[the remark this answers is not in the document]
Comment (TanDangNgoc) [reply src/containers/Home/component/SearchBox/index.tsx]: changing className due to BEM
image

File: src/containers/Home/index.tsx [hunk 2]
```diff
@@ -1,19 +1,45 @@
...
-          Welcome to Faba onboarding project - Job board
-        </p>
-      </header>
-    </div>
-  );
+    const [totalJobs, setTotalJobs] = useState<Jobs[]>([])
+    const [availableJobs, setAvailableJobs] = useState<number>(0)
+
+    useEffect(() => {
+        const handleGetData = async () => {
```
[the remark this answers is not in the document]
Comment (TanDangNgoc) [reply src/containers/Home/index.tsx:16]: Moving callAPI function to store
image
image

File: src/containers/Home/component/Result/Result.tsx [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+interface PropsBox {
+    title:string;
+    company: string;
+    available: boolean;
+    position: string;
+    location: string;
+    salary: number;
+}
+
+const Box:React.FC<PropsBox> = ({title, company, available, position, location, salary}) => {
```
[the remark this answers is not in the document]
Comment (TanDangNgoc) [reply src/containers/Home/component/Result/Result.tsx]: Changing list of props to Job props
image

File: src/containers/Home/component/Result/result.test.tsx [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+            Company: 'FabaTechnology',
+            Tags: 'IT, Front-end',
+            jobType: 'full-time',
+            Location: 'Ho chi Minh City',
+            Salary: 450,
+            Title: 'Looking for fresher ReactJs Developer'
+        }
+    ]
+    const tree = renderer.create(<Result totalJobs={fakeJobs}/>).toJSON()
+    expect(tree).toMatchSnapshot()
```
[the remark this answers is not in the document]
Comment (TanDangNgoc) [reply src/containers/Home/component/Result/result.test.tsx:53]: Testing data is available
image

File: src/containers/Home/component/Jobs.ts [hunk 5]
```diff
@@ -0,0 +1,11 @@
+export interface Jobs {
+    id: number;
+    Available: boolean;
```
[the remark this answers is not in the document]
Comment (TanDangNgoc) [reply src/containers/Home/component/Jobs.ts:3]: Adding key as string
image

File: src/containers/Home/index.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (hienntfaba) [reply src/containers/Home/index.tsx]: Good work!

File: src/containers/Home/component/Jobs.ts (hunk 5 above)
[the remark this answers is not in the document]
Comment (TanDangNgoc) [reply src/containers/Home/component/Jobs.ts]: Changing keyword
image

[the remark this answers is not in the document]
Comment (TanDangNgoc) [reply src/containers/Home/component/Jobs.ts]: camelCase keyword
image

File: src/containers/Home/component/SearchBox/index.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (TanDangNgoc) [reply src/containers/Home/component/SearchBox/index.tsx]: Edit className 
image

**Review by hienntfaba — APPROVED** [review]

**Pull request merged**